web/src/js/ducks/flows.ts

```diff
@@ -110,8 +113,15 @@ export default function reducer(
             return {
                 ...state,
                 selected: action.flowIds,
```

Comment:
Shouldn't this be `selected: [action.flowIds],`?

Reply:
`action.flowIds` is already an array of strings. Using `[action.flowIds]` would create an array containing an array of strings